Gemfile

```diff
@@ -1,12 +1,13 @@
 source "https://rubygems.org"
 
 ruby RUBY_VERSION
-DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+# DECIDIM_VERSION = { git: 'https://github.com/gencat/decidim', tag: 'v0.18-dev-gencat-features' }.freeze
+DECIDIM_VERSION = { path: '../decidim-fork' }.freeze
```

Comment:
This is intentional?

Reply:
Iep! @tramuntanal

Reply:
I may have introduced it by error when merging a stash. @isaacmg410 can you please confirm?

Reply:
Thanks @aitorlb you were right! fixed!